src/graph/query_graph.c

```diff
@@ -222,6 +228,55 @@ EntityType QueryGraph_GetEntityTypeByAlias(const QueryGraph *qg, const char *ali
 	return ENTITY_UNKNOWN;
 }
 
+// Returns if there was an update for an unkown label id.
+static bool _QGNode_Update(QGNode *node) {
+	bool updated = false;
+	if(node->labelID == GRAPH_UNKNOWN_LABEL) {
+		GraphContext *gc = QueryCtx_GetGraphCtx();
```

Comment:
No need to fetch `GraphContext` for each node, get it once and use for the entire node/edge update procedure